examples/mp-kv-sharding/src/main.rs

```diff
@@ -699,6 +705,294 @@ fn test_distribution_node_removed(config: &TestConfig) -> TestResult {
     check(&mut sys, &procs, &kv, false, true)
 }
 
+// MODEL CHECKING ------------------------------------------------------------------------------------------------------
+
+fn mc_check_query<S>(
+    sys: &mut System,
+    proc: S,
+    invariant: InvariantFn,
+    msg: Message,
+    start_states: Option<HashSet<McState>>,
+    max_depth: u64,
+) -> Result<McStats, String>
+where
+    S: Into<String>,
+{
+    let proc = proc.into();
+    let strategy_config = StrategyConfig::default()
+        .goal(goals::all_goals(vec![
+            goals::event_happened_n_times(LogEntry::is_mc_local_message_sent, 1),
+            goals::no_events(),
+        ]))
+        .invariant(invariants::all_invariants(vec![
+            invariant,
+            invariants::state_depth_current_run(max_depth),
+        ]))
+        .collect(collects::event_happened_n_times(LogEntry::is_mc_local_message_sent, 1));
+    let mut mc = ModelChecker::new::<Bfs>(sys, strategy_config);
```

Comment:
```suggestion
    let mut mc = ModelChecker::new::<Bfs>(sys);
```

перенести strategy_config в run

Reply:
перенес и адаптировал тесты